torch/package/importer.py

```diff
@@ -215,6 +215,14 @@ def get_source(self, module_name) -> str:
         module = self.import_module(demangle(module_name))
         return self.zip_reader.get_record(demangle(module.__file__)).decode('utf-8')
 
+    def get_resource_reader(self, fullname):
+        try:
+            package = self._get_package(fullname)
+        except ImportError:
+            return None
+        assert package.__loader__ is self
```

Comment:
In this case, should be return `None`. Otherwise It assert fail if some asks for resources for "sys", for instance.